jboss-kie-kieserver/configure.sh

```diff
@@ -61,3 +61,8 @@ chmod -R 777 /opt/kie/dependencies/jbpm-clustering
 # Create dir to remove JDBC driver
 mkdir ${JBOSS_HOME}/modules/system/layers/openshift 2&> /dev/null || true
 chown -R jboss:root ${JBOSS_HOME}/modules/system/layers/openshift
+
+# Dir for optional deps
+mkdir -p /opt/kie/dependencies/jbpm-kafka
+chown -R jboss:root /opt/kie/dependencies/jbpm-kafka
+chmod -R 777 /opt/kie/dependencies/jbpm-kafka
```

Comment:
please do not use 777, it should be 755.
also, please remove it from **chmod -R 777 /opt/kie/dependencies/jbpm-clustering**
and other places in this script.

Reply:
ok